backend/python/app/rest/__init__.py

```diff
@@ -1,14 +1,14 @@
 def init_app(app):
-    # auth {
-    from . import user_routes, auth_routes, entity_routes, documentation_routes
-    # } auth
-    # no-auth {
-    from . import entity_routes, documentation_routes
-    # } no-auth
+    from . import (
```

Comment:
Does this PR include removing **all** auth and file-storage tags from the Python backend? Wondering if this conflicts with what @matthewgeng is working on

Reply:
No, only files I had to modify